package.json

```diff
@@ -22,7 +24,7 @@
   "scripts": {
     "start": "npm run build",
     "build": "npm run build:scripts && npm run build:styles",
-    "build:scripts": "browserify src/scripts/main.js -t [ babelify --presets es2015 ] -o dist/scripts/main.js",
+    "build:scripts": "browserify src/scripts/main.js -t [ babelify --presets es2015 ] -o dist/scripts/main.js && browserify src/scripts/background.js -t [ babelify --presets es2015 ] -o dist/scripts/background.js",
```

Comment:
I don't think most browsers need babel anymore, this can be simplified to

```json
    "build:scripts": "browserify src/scripts/main.js -o dist/scripts/main.js && browserify src/scripts/background.js -o dist/scripts/background.js",
```